addons/api/addon/models/worker.js

```diff
@@ -34,14 +34,21 @@ export default class WorkerModel extends GeneratedWorkerModel {
   }
 
   /**
-   * Returns the number of canonical tags present on the worker.
+   * Returns the number of tags present on the worker.
    * @type {number}
    */
   get tagCount() {
-    if (!this.canonical_tags) {
+    if (
+      !Object.keys(this.config_tags).length &&
+      !Object.keys(this.api_tags).length
```

Comment:
It seems like it's erroring out trying to goto the workers list page:

<img width="501" alt="image" src="https://github.com/user-attachments/assets/e9c36794-168b-4728-8b4e-e4a429d9f41a">

Reply:
It doesn't look like it correctly normalizes the response for list responses right now but also, if these were getting normalized correctly, you shouldn't even need this if check.